Mathlib/Tactic/Linter/TextBased.lean

```diff
@@ -190,6 +246,15 @@ def parse?_errorContext (line : String) : Option ErrorContext := Id.run do
             some (StyleError.broadImport BroadImports.TacticFolder)
           else
             some (StyleError.broadImport BroadImports.Lake)
+        | "ERR_UNICODE" =>
+            if let some str := errorMessage.get? 2 then
+              if let some c := str.get? ⟨1⟩ then
+                some (StyleError.unwantedUnicode c)
+              else none
+            else none
+        | "ERR_UNICODE_VARIANT" =>
+          -- some (StyleError.unicodeVariant _ _ _)
+          none --TODO
```

Comment:
I think the idea here is to parse back the text representation of the linter error into an `ErrorContext` object. I don't actually know what it's used for. If I understand it correctly, it should be the inverse of `outputMessage`, which actually isn't strictly possible the way that we print the error (we ignore the second character, for example). 

Probably this doesn't matter and we just "best-effort" parse the error message.  I could do it if you like...

Reply:
Did this.